edkahara/iReporter
# Ft create v2 get reports endpoints #162423172

__What does this PR do__
Create get all reports endpoints.

__Description of tasks to be completed.__
Create api endpoints that facilitates the getting of all reports. Test the api endpoints.

__How should it be manually tested?__
Clone the project. Check out the ft-create-v2-get-reports-endpoints-#162423172 branch. Start the environment using `env\Scripts\activate` and `env.bat` to set up environment variables. Run flask run on the terminal and go to Postman. To perform the tests run nosetests.

__Relevant pivotal tracker stories.__
#162423172
